functionaltests/internal/kbclient/client.go

```diff
@@ -30,17 +30,13 @@ import (
 	"time"
 )
 
-func New(kibanaURL, apiKey string, username, password string) (*Client, error) {
+func New(kibanaURL, username, password string) (*Client, error) {
```

Comment:
I think it's ok to keep using only username and password also when we decommission 7.x line, so I'm 100% onboard with this change and not making this 7.x specific.